### PR TITLE
readme rewrite

### PR DESCRIPTION
**Pull request by tobybellwood**
A rewrite of the readme for Lagoon

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,31 +1,159 @@
...
+
+For more information on developing or contributing to Lagoon, head to https://docs.lagoon.sh/lagoon/contributing-to-lagoon
+
+For more information on installing and administering Lagoon, head to https://docs.lagoon.sh/lagoon/administering-lagoon
+
+## Lagoon architecture
+
+Lagoon is comprised of two main components, Lagoon Core and Lagoon Remote, and a number of other third-party services, Operators and Controllers.  In a full production setting, it is envisaged that Lagoon Core and Remote are installed into different Kubernetes Clusters (one Core can serve many Remotes), but they can also be installed into the same cluster if required.  Additionally, in advanced situations, a Lagoon Remote can contain multiple build-deploy controllers, connected to different Lagoon Cores (but this is only really recommended for testing purposes).
```
Comment (Schnitzel) [inline README.md]: I would not mention the Additionally piece, it's super super edgy and makes the whole thing here more complex to understand

File: README.md [hunk 2]
```diff
@@ -1,31 +1,159 @@
...
+
+All communication between Lagoon Core and Lagoon Remote happens via RabbitMQ message queues, GraphQL or REST APIs.  Lagoon Core does not require administrator-level access to Lagoon Remote clusters.
```
Comment (Schnitzel) [inline README.md]: I would change this in something like this:

```The communication between core and remote happens exclusively via RabbiqMQ, which also the connection is initiated from the lagoon remote to the lagoon core, which makes it possible to run lagoon remote on kubernetes clusters that are not accessible from the internet.```

Plus I would write:
```
Lagoon Core does not require administrator-level access to Kubernetes Clusters running Lagoon Remote.
```

File: README.md [hunk 3]
```diff
@@ -1,31 +1,159 @@
...
+  - [logs2rocketchat](https://github.com/amazeeio/lagoon/tree/main/services/logs2rocketchat) (the service that pushes build notifications to a nominated Rocket Chat channel)
+  - [logs2microsoftteams](https://github.com/amazeeio/lagoon/tree/main/services/logs2microsoftteams) (the service that pushes build notifications to a nominated Microsoft Teams channel)
+- Other Services
+  - [ui](https://github.com/amazeeio/lagoon/tree/main/services/ui) (the main user interface and dashboard for Lagoon)
+  - [lagoon-cli](https://github.com/amazeeio/lagoon-cli) (the command-line interface for managing sites on Lagoon)
+  - [drush-alias](https://github.com/amazeeio/lagoon/tree/main/services/drush-alias) (provides Drupal developers with an automated alias service for Drush)
+
+### Lagoon Remote
+
+All the services that are used to provision, deploy and maintain sites hosted on Lagoon live here.  These services are mostly comprised of third-party tools, developed external to Lagoon itself.  Installation is via a [Helm Chart](https://github.com/uselagoon/lagoon-charts/tree/main/charts/lagoon-remote)
```
Comment (Schnitzel) [inline README.md]: I would suggest:
```
All the services that are used to provision, deploy and maintain sites hosted by Lagoon on Kubernetes live here.```

File: README.md [hunk 4]
```diff
@@ -1,31 +1,159 @@
...
+
+Lagoon has a comprehensive [test suite](https://github.com/amazeeio/lagoon/tree/main/tests/tests), designed to cover most end-user scenarios.  The testing is automated in Ansible, and runs in a Jenkins install, but can also be run locally in a self-contained cluster.  The testing provisions a standalone Lagoon cluster, running on KinD, comprising Lagoon Core, Lagoon Remote, a Registry and a set of Managed Databases, and runs test deployments and scenarios for a range of Node.js, Drupal, Python and NGINX projects, all built using the latest Lagoon Images.
+
+
+
+## Other Lagoon components
+
+Here are a number of other repositories, tools and components used in Lagoon
+
+### [Lagoon Images](https://github.com/uselagoon/lagoon-images)
```
Comment (Schnitzel) [inline README.md:112]: maybe we want to explain the concept of base images a little bit more here? basically that they are suggest to be used by application developers as a base for the images they build to run the application with. That Lagoon actually does not require these base images to deploy an application into a kubernetes cluster, but many functionalities of Lagoon might not work if the base images are not used.

File: README.md (hunk 2 above)
Comment (tobybellwood) [reply README.md]: I've reworded this a bit more

File: README.md [hunk 5]
```diff
@@ -1,31 +1,159 @@
...
-  <img src="https://raw.githubusercontent.com/amazeeio/lagoon/master/docs/images/lagoon-logo.png" alt="The Lagoon logo is a blue hexagon split in two pieces with an L-shaped cut" width="40%">
+ <img src="./Lagoon_OG.png" alt="The Lagoon logo is a blue hexagon split in two pieces with an L-shaped cut" width="40%">
 </p>
 
-# amazee.io Lagoon - Docker Build and Deploy System for OpenShift & Kubernetes
+# Lagoon - the developer-focused application delivery platform for Kubernetes
+
+Lagoon solves what developers are dreaming about: A system that allows developers to locally develop their code and their services with Docker and run the exact same system in production. The same container images, the same service configurations and the same code.
+
+> As an application delivery **platform**, the primary focus of Lagoon is as a tool for the deployment, management, security and operation of multiple applications, utilising cloud-native technologies, but with minimal imposition or knowledge expectation on the developers of those applications.
```
Comment (AlannaBurke) [inline README.md]: This paragraph is a little complex, I ran it through Hemingway and tried to simplify it - thoughts?
> Lagoon is an application delivery **platform**. Its primary focus   is as a tool for the deployment, management, security and operation of many applications. Lagoon uses cloud-native technologies, but with little knowledge needed by the application developers.

File: README.md [hunk 6]
```diff
@@ -1,31 +1,159 @@
...
+- Locally (for evaluation, testing, debugging or development) using [Helm](https://helm.sh/) charts and [kind](https://kind.sigs.k8s.io/)
+- Into your managed Kubernetes cloud provider of choice - it's running in production on [Amazon Elastic Kubernetes Service](https://aws.amazon.com/eks/), [Azure Kubernetes Service](https://azure.microsoft.com/en-au/services/kubernetes-service/), and [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine/), but we are confident that it will also run on any other managed Kubernetes service.
+
+For more information on developing or contributing to Lagoon, head to https://docs.lagoon.sh/lagoon/contributing-to-lagoon
+
+For more information on installing and administering Lagoon, head to https://docs.lagoon.sh/lagoon/administering-lagoon
+
+## Lagoon architecture
+
+Lagoon is comprised of two main components, Lagoon Core and Lagoon Remote, and a number of other third-party services, Operators and Controllers. In a full production setting, it is envisaged that Lagoon Core and Remote are installed into different Kubernetes Clusters (one Core can serve many Remotes), but they can also be installed into the same cluster if required.
```
Comment (AlannaBurke) [inline README.md]: Simplifying language: 
> Lagoon is made of two main components: Lagoon Core and Lagoon Remote. It's also built on several other third-party services, Operators and Controllers. In a full production setting, we generally  install Lagoon Core and Remote into different Kubernetes Clusters. One Core can serve many Remotes. They can also be installed into the same cluster if required.

File: README.md [hunk 7]
```diff
@@ -1,31 +1,159 @@
...
+
+To enhance security, Lagoon Core does not require administrator-level access to the Kubernetes clusters running Lagoon Remote, and all inter-cluster communication happens exclusively via RabbitMQ, hosted in Lagoon Core, and consumed (and published back to) by Lagoon Remote.  This means that Lagoon Remotes can be managed by different teams, in different locations, and even behind firewalls or inaccessible from the internet.
+
+Lagoon services are mostly built in Node.js, with more recent development in occuring in Go, and most of the automation and scripting components in Bash.
+
```
Comment (AlannaBurke) [inline README.md:33]: simplifying
> To enhance security, Lagoon Core does not need administrator-level access to the Kubernetes clusters that are running Lagoon Remote.  All inter-cluster communication happens only via RabbitMQ. This is hosted in Lagoon Core, and consumed (and published back to) by Lagoon Remote.  This allows Lagoon Remotes to be managed by different teams, in different locations -  even behind firewalls or inaccessible from the internet.

> Lagoon services are mostly built in Node.js. More recent development occurs in Go, and most of the automation and scripting components are in Bash.

File: README.md [hunk 8]
```diff
@@ -1,31 +1,159 @@
...
+  - PostgreSQL (self managed or via [Amazon RDS for PostgreSQL](https://aws.amazon.com/rds/postgresql/), [Amazon Aurora PostgreSQL](https://aws.amazon.com/rds/aurora/postgresql-features/), [Azure Database for PostgreSQL](https://docs.microsoft.com/en-us/azure/postgresql), [Cloud SQL for PostgreSQL](https://cloud.google.com/sql/docs/postgres) )
+
+  - MongoDB (self managed, or via [Amazon DocumentDB](https://aws.amazon.com/documentdb/), [Azure Cosmos DB](https://azure.microsoft.com/en-au/services/cosmos-db/) )
+
+### Testing
+
+Lagoon has a comprehensive [test suite](https://github.com/amazeeio/lagoon/tree/main/tests/tests), designed to cover most end-user scenarios. The testing is automated in Ansible, and runs in a Jenkins install, but can also be run locally in a self-contained cluster. The testing provisions a standalone Lagoon cluster, running on KinD, comprising Lagoon Core, Lagoon Remote, a Registry and a set of Managed Databases, and runs test deployments and scenarios for a range of Node.js, Drupal, Python and NGINX projects, all built using the latest Lagoon Images.
+
```
Comment (AlannaBurke) [inline README.md]: simplifying
> Lagoon has a comprehensive [test suite](https://github.com/amazeeio/lagoon/tree/main/tests/tests), designed to cover most end-user scenarios. The testing is automated in Ansible, and runs in a Jenkins install, but can also be run locally in a self-contained cluster. The testing provisions a standalone Lagoon cluster, running on KinD (Kubernetes in Docker). This cluster is made of Lagoon Core, Lagoon Remote, a registry and a set of managed databases. It runs test deployments and scenarios for a range of Node.js, Drupal, Python and NGINX projects, all built using the latest Lagoon images.

**Review by AlannaBurke — CHANGES_REQUESTED** [review]

**Pull request merged**